plugins/kubernetes-react/api-report.md

```diff
@@ -0,0 +1,448 @@
+}: PendingPodContentProps) => React_2.JSX.Element;
+
+// @public
+export interface PendingPodContentProps {
+  // (undocumented)
+  pod: Pod;
+}
+
+// @public
+export interface PodAndErrors {
+  // (undocumented)
+  clusterName: string;
+  // (undocumented)
+  errors: DetectedError[];
+  // (undocumented)
+  pod: Pod;
+}
+
+// @public (undocumented)
+export type PodColumns = 'READY' | 'RESOURCE';
+
+// @public
+export const PodDrawer: ({
+  podAndErrors,
+  open,
+}: PodDrawerProps) => React_2.JSX.Element;
+
+// @public
+export interface PodDrawerProps {
+  // (undocumented)
+  open?: boolean;
+  // (undocumented)
+  podAndErrors: PodAndErrors;
+}
+
+// @public
+export const PodExecTerminal: (
+  props: PodExecTerminalProps,
+) => React_2.JSX.Element;
+
+// @public
+export const PodExecTerminalDialog: (
+  props: PodExecTerminalProps,
+) => React_2.JSX.Element;
+
+// @public
+export interface PodExecTerminalProps {
+  // (undocumented)
+  clusterName: string;
+  // (undocumented)
+  containerName: string;
+  // (undocumented)
+  podName: string;
+  // (undocumented)
+  podNamespace: string;
+}
+
+// @public
+export const PodLogs: React_2.FC<PodLogsProps>;
+
+// @public
+export const PodLogsDialog: ({
+  containerScope,
+}: PodLogsDialogProps) => React_2.JSX.Element;
+
+// @public
+export interface PodLogsDialogProps {
+  // (undocumented)
+  containerScope: ContainerScope;
+}
+
+// @public
+export interface PodLogsOptions {
+  // (undocumented)
+  containerScope: ContainerScope;
+  // (undocumented)
+  previous?: boolean;
+}
+
+// @public
+export interface PodLogsProps {
+  // (undocumented)
+  containerScope: ContainerScope;
+  // (undocumented)
+  previous?: boolean;
+}
+
+// @public
+export interface PodScope {
+  // (undocumented)
+  clusterName: string;
+  // (undocumented)
+  podName: string;
+  // (undocumented)
+  podNamespace: string;
+}
+
+// @public (undocumented)
+export const PodsTable: ({
+  pods,
+  extraColumns,
+}: PodsTablesProps) => React_2.JSX.Element;
+
+// @public (undocumented)
+export type PodsTablesProps = {
+  pods: Pod_2 | V1Pod[];
+  extraColumns?: PodColumns[];
+  children?: React_2.ReactNode;
+};
+
+// @public (undocumented)
+export const READY_COLUMNS: PodColumns;
+
+// @public (undocumented)
+export const RESOURCE_COLUMNS: PodColumns;
+
+// @public
+export const ResourceUtilization: ({
+  compressed,
+  title,
+  usage,
+  total,
+  totalFormatted,
+}: ResourceUtilizationProps) => React_2.JSX.Element;
+
+// @public
+export interface ResourceUtilizationProps {
+  // (undocumented)
+  compressed?: boolean;
+  // (undocumented)
+  title: string;
+  // (undocumented)
+  total: number | string;
+  // (undocumented)
+  totalFormatted: string;
+  // (undocumented)
+  usage: number | string;
+}
+
+// @public (undocumented)
+export const ServicesAccordions: ({}: ServicesAccordionsProps) => React_2.JSX.Element;
+
+// @public (undocumented)
+export type ServicesAccordionsProps = {};
+
+// @public
+export const useEvents: ({
```

Comment:
This naming is quite generic 😅

Reply:
Mind if I change to be more descriptive in a follow up?

Reply:
Yep np!